### PR TITLE
Replacing links to code. and svn.enthought.com

### PR DESCRIPTION
**Pull request by kitchoi**
Hunt down links to the code.enthought.com and svn.enthought.com domain.

Some of which are in scripts' docstrings that are invisible on the actual documentation website.

And I don't know what to do with these:
```
./docs/source/mayavi/installation.rst:.. _ETS: http://code.enthought.com/
./docs/source/mayavi/overview.rst:.. _ETS: http://code.enthought.com/projects/tool-suite.php
./examples/mayavi/mlab/chemistry.py:    'http://code.enthought.com/projects/mayavi/data/h2o-elf.cube'
./docs/source/mayavi/auto/chemistry.py:   'http://code.enthought.com/projects/mayavi/data/h2o-elf.cube'
```

Also don't know what to do with these:
```
./docs/API.txt:   https://svn.enthought.com/enthought/wiki/EnthoughtVersionNumbers
./mayavi/scripts/mayavi2.py:Mayavi2 wiki page: http://svn.enthought.com/enthought/wiki/MayaVi
```

I can't guarantee that I have done all of them, but this is a step forward.

@jdmarch can you take a look please?

### CODE REVIEW

**Comment by corranwebster** [comment]
The link `./docs/API.txt:   https://svn.enthought.com/enthought/wiki/EnthoughtVersionNumbers` should point to the github repo (possibly to the releases sub-page https://github.com/enthought/mayavi/releases).

The link `./mayavi/scripts/mayavi2.py:Mayavi2 wiki page: http://svn.enthought.com/enthought/wiki/MayaVi` should point to the Mayavi documentation http://docs.enthought.com/mayavi/mayavi/

**Comment by kitchoi** [comment]
Thanks @corranwebster! just updated those links

**Comment by corranwebster** [comment]
Having a look, I think that the other ones have to wait until we know what we are going to do with code.enthought.com.  The datasets that are stored there could be moved into the github repo and the download links updated to grab the appropriate url; or we could add them to the documentation and download them from docs.enthought.com, but either way we have to keep the datasets at code.enthought.com for a while because people will still have old versions of Mayavi around for a while.

File: README-tvtk.txt [hunk 1]
```diff
@@ -64,16 +55,12 @@ Use a similar line for your particular shell.
...
-The bug tracker is available as part of the trac interface here:
-
- https://svn.enthought.com/enthought/
+The bug tracker is available here:
 
-To submit a bug you will necessarily have to register at the site.  Click on
-the "register" link at the top right on the above page to register.  Or login
-if you already have registered.  Once you are registered you may file a bug by
-creating a new ticket.
+ https://github.com/enthought/mayavi
```
Comment (corranwebster) [inline README-tvtk.txt]: May as well link directly to https://github.com/enthought/mayavi/issues

File: docs/source/mayavi/data.rst [hunk 2]
```diff
@@ -518,7 +518,7 @@ objects or data files for Mayavi and TVTK.
 
  * Presentation on TVTK and Mayavi2 for course at IIT Bombay
 
-   https://svn.enthought.com/enthought/attachment/wiki/MayaVi/tvtk_mayavi2.pdf
+   In source: ./docs/pdf/tvtk_mayavi2.pdf
```
Comment (corranwebster) [inline docs/source/mayavi/data.rst]: Can link directly to github: https://github.com/enthought/mayavi/raw/master/docs/pdf/tvtk_mayavi2.pdf

File: examples/mayavi/interactive/adjust_cropping_extents.py [hunk 3]
```diff
@@ -14,9 +14,11 @@
 are modified to propagate them to the filter. For more information
 on creating GUIs with Traits:
 
-    http://code.enthought.com/projects/traits/docs/html/tutorials/traits_ui_scientific_app.html
+    http://docs.enthought.com/traitsui
```
Comment (corranwebster) [inline examples/mayavi/interactive/adjust_cropping_extents.py]: This tutorial is now part of the TraitsUI documentation: http://docs.enthought.com/traitsui/tutorials/traits_ui_scientific_app.html

Comment (corranwebster) [inline examples/mayavi/interactive/adjust_cropping_extents.py]: But it's worthwhile having a link to the main TraitsUI page as well.

**Comment by corranwebster** [comment]
Other than the above notes, looks good to me.

File: examples/mayavi/interactive/adjust_cropping_extents.py (hunk 3 above)
Comment (kitchoi) [inline examples/mayavi/interactive/adjust_cropping_extents.py]: http://docs.enthought.com/traitsui/tutorials/traits_ui_scientific_app.html is linked via the third link in this docstring: https://support.enthought.com/hc/en-us/articles/204469620-Introductory-materials-for-Traits-and-Traits-UI  so I decided to skip that.

**Comment by kitchoi** [comment]
Thanks @corranwebster!  I have made the changes

**Comment by prabhuramachandran** [comment]
@kitchoi -- this is fantastic, thank you so much!

**Pull request merged**